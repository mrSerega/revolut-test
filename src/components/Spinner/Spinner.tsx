import React from 'react';

import './Spinner.css'

export interface SpinnerProps {

}

export class Spinner extends React.Component<SpinnerProps> {
    readonly blockName = 'spinner'

    render() {
        return <svg height="56" width="56" className={this.blockName}>
            <circle
                cx="28"
                cy="28"
                r="24"
                stroke="white"
                stroke-width="4"
                fill="transparent"
                stroke-dasharray="0 10" stroke-linecap="round"
            />
        </svg>
    }
}