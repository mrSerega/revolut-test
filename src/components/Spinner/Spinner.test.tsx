import React from 'react';
import renderer from 'react-test-renderer';
import { Spinner } from './Spinner';

test('Spinner', () => {
    const cmp = renderer.create(
        <Spinner/>
    )
    const tree = cmp.toJSON()
    expect(tree).toMatchSnapshot()
})