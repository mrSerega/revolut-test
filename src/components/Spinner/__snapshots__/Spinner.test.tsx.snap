// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner 1`] = `
<svg
  className="spinner"
  height="56"
  width="56"
>
  <circle
    cx="28"
    cy="28"
    fill="transparent"
    r="24"
    stroke="white"
    strokeDasharray="0 10"
    strokeLinecap="round"
    strokeWidth="4"
  />
</svg>
`;
